port_ocean/core/handlers/webhook/abstract_webhook_processor.py

```diff
@@ -85,6 +88,13 @@ async def after_processing(self) -> None:
         """Hook to run after processing the event"""
         pass
 
+    async def process_data(self, kind: str, data: list[RAW_ITEM]) -> None:
+        """Process the data for the given kind:
+        1) map data to port entities
+        2) update data in port - upsert and delete entities accordingly
+        """
```

Comment:
I wonder if the description of what is being done in `process_data` rather than `on_live_event`